docs/SNAPTELD.md

```diff
@@ -44,6 +44,8 @@ $ snapteld [global options] command [command options] [arguments...]
 --control-listen-port value                  Listen port for control RPC server (default: 8082) [$SNAP_CONTROL_LISTEN_PORT]
 --control-listen-addr value                  Listen address for control RPC server [$SNAP_CONTROL_LISTEN_ADDR]
 --temp_dir_path value                        Temporary path for loading plugins [$SNAP_TEMP_DIR_PATH]
+--tls-cert value                             A path to PEM-encoded certificate to use for TLS channels
+--tls-key value                              A path to PEM-encoded private key file to use for TLS channels
```

Comment:
Those options should have more documentation, so that users know how to enable secure plugin communication.